### DISCUSSION TITLE
Save all without any propmt

### DISCUSSION THREAD

**Discussion by whatthefurk** [tannerhelland/PhotoDemon#603] [Q&A] [answered]
Hi!
PhotoDemon is a great software for what I need to do with my image files. However, I am unable to accomplish the following: Let's say I open 10 images at once and edit every image. Now I want to save and close all images at once without getting a prompt for every image (right now, "JPEG options" pops up every time, but since I'd like to save all images with the standard options, e.g. quality 92, compression optimized baseline, it's not necessary for me and quite time consuming for +200 files). Is there a way I can achieve not getting prompted?
Thanks!

**Comment by tannerhelland** [comment, marked as answer]
Hi @whatthefurk .  Thanks for mentioning this.

Another user requested this feature, and it's on my TODO list.  I'll track progress here once work starts:

https://github.com/tannerhelland/PhotoDemon/issues/566
